main/updater/manualCheck.ts

```diff
@@ -0,0 +1,65 @@
+import log from 'electron-log'
+import https from 'https'
+import semver from 'semver'
+import { VersionUpdate } from '.'
```

Comment:
Make it more obvious you're just importing the type here
`import type { VersionUpdate } from '.'`